sycl/source/detail/jit_compiler.cpp

```diff
@@ -808,7 +808,7 @@ jit_compiler::fuseKernels(QueueImplPtr Queue,
       std::move(ArgsStorage), std::move(AccStorage),
       std::move(RawExtendedMembers), std::move(Requirements), std::move(Events),
       std::move(FusedArgs), FusedKernelInfo.Name, OSUtil::DummyModuleHandle, {},
-      {}, CG::CGTYPE::Kernel));
+      {}, CG::CGTYPE::Kernel, {}));
```

Comment:
Would it make sense to try and merge the cache/SLM property from the input kernels? 

For example, if some kernels specify the same cache/SLM property and some other do not specify any at all? 

Also note that with local internalization, the fusion process might introduce additional local memory, tipping the balance to prefer more SLM.

Reply:
Good question, I am not sure if there is a good way to merge properties for fused kernels.
Default mode for fused kernel seems ok to me.

@jandres742 @smaslov-intel Could you provide your input please?

Reply:
Implemented per suggestion from @smaslov-intel.